### DISCUSSION TITLE
Cannot find name 'RequestInfo'. Did you mean 'RequestInit'?(2552)

### DISCUSSION THREAD

**Discussion by PPetterssonSO** [SuperOffice/typescript-for-superoffice#18] [General]
Copy/pased the httpRequest sample, and, it complains about RequestInfo and text()
It still works just fine, but shows red underlines for the above

**Comment by ejfasting** [comment]
Hi @Nawksen, 

Thank you for bringing it to our attention! 
I will create a bugreport on this, as its 'just' the types missing. The codesample should still run fine even though the intellisense gives the wrong impression. 

/Eivind
